_tuning-your-cluster/availability-and-recovery/remote-store/snapshot-interoperability.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Shallow snapshots
+nav_order: 15
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Shallow snapshots
+
+Shallow copy snapshots allow you to reference data from an entire remote-backed segment instead of storing all of the data from the segment in a snapshot. This makes accessing segment data faster than using normal snapshots because segment data is not stored in the snapshot repository.
```

Comment:
Shallow copy snapshots allows you to reference data directly from remote store instead of storing all of the segment data again in snapshot repository. These snapshots gets created faster than normal snapshots because segment data is not stored in the snapshot repository.